content/en/docs/concepts/services-networking/ingress.md

```diff
@@ -7,7 +7,7 @@ weight: 40
 ---
 
 <!-- overview -->
-{{< feature-state for_k8s_version="v1.19" state="stable" >}}
+{{< feature-state for_k8s_version="v1.1" state="bate" >}}
```

Comment:
I think this is stable already at 1.19 why would we change this to beta on 1.1? Furthermore there’s a typo there.